agreement/migrations/0006_add_hopkinsIT_agreement.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright (C) 2017 Semester.ly Technologies, LLC
+Semester.ly is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+Semester.ly is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+"""
+
+from __future__ import unicode_literals
+
+from django.db import migrations
+from agreement.models import Agreement
+import datetime
+
+
+def add_hopkinsIT_agreement(apps, schema_editor):
+
+    agreement, created = Agreement.objects.get_or_create(
+        url="\\notice",
+        last_updated=datetime.datetime.now(),
```

Comment:
should we hardcode this to a date instead of just doing now? otherwise say you move to a new server and run migrations, it will look like there's a new agreement when there isn't actually